docs/source/models.rst

```diff
@@ -1027,6 +1027,56 @@ method to load MLflow Models with the ``catboost`` model flavor in native CatBoo
 
 For more information, see :py:mod:`mlflow.catboost`.
 
+``CatBoost`` pyfunc usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a CatBoost Classifier model, an example configuration for the pyfunc predict() method is:
+
+.. code-block:: py
+
+	import mlflow
```

Comment:
Looks like you're using tabs. Can we replace them with spaces?

Reply:
sure